<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab Helper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="11">
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="btn_startMeasurement">
      <property name="text">
       <string>st&amp;art</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="txt_updateInterval">
      <property name="text">
       <string>interval (sec):</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QTimeEdit" name="time_timeRunning">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>hh:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QDoubleSpinBox" name="dsp_updateInterval">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <double>0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>4000.000000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="btn_stopMeasurement">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>st&amp;op</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QPushButton" name="btn_addUiCharWindows_new">
      <property name="text">
       <string>add UI-&amp;Char Windows</string>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QPushButton" name="btn_addWindow">
      <property name="text">
       <string>add &amp;Window</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="lbl_updateStatus">
      <property name="font">
       <font>
        <italic>true</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red</string>
      </property>
      <property name="text">
       <string> paused </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="btn_emergencyStop">
      <property name="styleSheet">
       <string notr="true">background-color: yellow</string>
      </property>
      <property name="text">
       <string>EMERGENCY STOP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="act_logValues"/>
    <addaction name="act_logInfos"/>
    <addaction name="separator"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="act_addWindow"/>
    <addaction name="act_tileWindows"/>
    <addaction name="act_cascadeWindows"/>
    <addaction name="act_addUiCharWindows_old"/>
    <addaction name="act_addUICharWindows_new"/>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuWindows"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_addWindow">
   <property name="text">
    <string>&amp;Add Window</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="act_logValues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Log values</string>
   </property>
  </action>
  <action name="act_addUiCharWindows_old">
   <property name="text">
    <string>Add UI-Char Windows (old)</string>
   </property>
  </action>
  <action name="act_logInfos">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log infos</string>
   </property>
  </action>
  <action name="act_addUICharWindows_new">
   <property name="text">
    <string>Add UI-Char Windows new</string>
   </property>
  </action>
  <action name="act_cascadeWindows">
   <property name="text">
    <string>Cascade Windows</string>
   </property>
  </action>
  <action name="act_tileWindows">
   <property name="text">
    <string>Tile Windows</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
